server/routerlicious/packages/services-shared/src/runner.ts

```diff
@@ -63,11 +65,14 @@ export function runService<T extends IResources>(
     runningP.then(
         () => {
             logger?.info("Exiting");
+            Lumberjack.info("Exiting");
```

Comment:
Thanks for this PR @TriDucNgo! I was thinking this `runService` method would be a great candidate for a metric. This is totally optional, and if you prefer we can leave things as they are now in your PR. But, if you are feeling like you want to add metrics in the code, we could create a new Lumber metric right above 
```typescript
const runningP = run(config, resourceFactory, runnerFactory, logger);
```
We would either mark the metric as success in the `.then() ()=>`, or fail it in the `(error) =>` flow. One thing we could use this metric for is container restarts, and we could create alerts for that.

Reply:
sure, that makes sense. We can add that in a separate PR.

Reply:
Once we add the metric, we can probably remove the logs too

Reply:
Indeed, that's my plan. I can work on that once @TriDucNgo completes this PR.